layouts/default/alice/info.json

```
@@ -0,0 +1,83 @@
+{
+    "keyboard_name": "Alice layout",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 18.35,
+    "height": 5.1,
+    "layouts": {
+        "LAYOUT_alice": {
+            "layout": [
+                {"x":0.4, "y":0},
+                {"x":3.55, "y":0},
+                {"x":13.75, "y":0},
+
+                {"x":1.55, "y":0.1},
+                {"x":2.55, "y":0.1},
+                {"x":4.55, "y":0.1},
+                {"x":5.55, "y":0.1},
+                {"x":6.55, "y":0.1},
+                {"x":7.55, "y":0.1},
+                {"x":9.75, "y":0.1},
+                {"x":10.75, "y":0.1},
+                {"x":11.75, "y":0.1},
+                {"x":12.75, "y":0.1},
+                {"x":14.75, "y":0.1},
+                {"x":15.75, "y":0.1, "w":2},
```

Fix to the key object ordering.

```suggestion
                {"x":0.4, "y":0},
                {"x":1.55, "y":0.1},
                {"x":2.55, "y":0.1},
                {"x":3.55, "y":0},
                {"x":4.55, "y":0.1},
                {"x":5.55, "y":0.1},
                {"x":6.55, "y":0.1},
                {"x":7.55, "y":0.1},
                {"x":9.75, "y":0.1},
                {"x":10.75, "y":0.1},
                {"x":11.75, "y":0.1},
                {"x":12.75, "y":0.1},
                {"x":13.75, "y":0},
                {"x":14.75, "y":0.1},
                {"x":15.75, "y":0.1, "w":2},
```

---

Does this fix the fast input mode issue? I tried repro'ing the issue but couldn't see anything wrong with it

---

Thanks I see now!